File: day 36/classwork/cw5.py
# 5) უკვე შექმნილ Dictionary-დან, შეცვალე key ისე, რომ value იგივე დარჩეს და დაბეჭდე Dictionary საბოლოო   სახით (გააკეთეთ pop-ით).

my_dict = {
    "name": "Ana",
    "age": 25,
    "city": "Tbilisi",
    "favorite_color": "blue"
}

# შეცვალე key "favorite_color" -> "color"
my_dict["color"] = my_dict.pop("favorite_color")

# დაბეჭდე საბოლოო Dictionary
print(my_dict)

File: day 60/classwork/cw5.js
const person = {
    name: "gio",
    age: 14,
    მისალმება: function () {
        console.log ("ჩემი სახელია", this.name + "და ვარ ", this.age + "წლის!")
    }
};

person.მისალმება ()